app/import.go

```diff
@@ -155,6 +168,16 @@ func (a *App) bulkImport(c *request.Context, fileReader io.Reader, dryRun bool,
 			return model.NewAppError("BulkImport", "app.import.bulk_import.json_decode.error", nil, err.Error(), http.StatusBadRequest), lineNumber
 		}
 
+		if len(attachedFiles) > 0 && line.Post != nil && line.Post.Attachments != nil && len(*line.Post.Attachments) > 0 {
```

Comment:
I am thinking the `len(*line.Post.Attachments) > 0` is redundant.

Reply:
Sure is, good catch.